src/pages/shikshagraha-repository/listing/Header.jsx

```
@@ -32,18 +32,18 @@ export default function Header({ isHeroSection = true, isBackButton = false, onS
 
   return (
     <>
-      <header className={`flex flex-col px-7 pt-4 pb-4 md:pb-8 w-full bg-white rounded-[16px] ${!isMobile ? "shadow-[0px_0px_4px_rgba(0,0,0,0.2)]" : ""} ${isMobile && isBackButton ? "items-start" : "items-end"}`}>
+      <header className={`flex flex-col p-[25px] md:pb-8 w-full bg-white rounded-[16px] ${!isMobile ? "shadow-[0px_0px_4px_rgba(0,0,0,0.2)]" : ""} ${isMobile && isBackButton ? "items-start" : "items-end"}`}>
         {(isHeroSection || !isMobile) && <div className="w-full">
           <>
             <style
               type="text/css"
               dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)